src/commons/Config.ts

```diff
@@ -18,10 +19,18 @@ class Config {
     this.setConfig({});
   }
 
-  public setConfig(options: ConfigOptions) {
+  public async setConfig(options: ConfigOptions) {
     const {usePlatformColors = false, appScheme = 'light'} = options;
     this.usePlatformColors = usePlatformColors;
-    this.appScheme = appScheme;
+    this.appScheme = await AsyncStoragePackage?.getItem?.('appScheme') || appScheme;
```

Comment:
I would also add an option called "useLocalScheme", so not all users have to await the result of async storage if they don't care for it.